ebookstore_flask/routes/staff_order.py

```diff
@@ -91,28 +94,44 @@ def finished():
 def returned():
    return index("returned")
 
+
 @staff_order.route('/<path:current_path>/find', methods=['POST'])
 def filter_by(current_path):
-   user_input = request.form['user_input']
+      user_input = request.form.get('user_input', "").strip() 
```

Comment:
i think for the find part, we can use javascript to do that, because if you using route it will be resulted like this provided image

![image](https://github.com/user-attachments/assets/087a08f9-1e50-4776-a92c-04f68d0731b8)

and the result also not found

![image](https://github.com/user-attachments/assets/633c564e-cb95-49bd-8158-c3adbee28f46)

but if you guys didnt know how to write javascript, i can help you guys later